modulesets/gtk-osx-network.modules

```diff
@@ -45,7 +45,9 @@
   <autotools id="openssl" autogen-sh="Configure" autogenargs="shared"
              autogen-template="%(srcdir)s/%(autogen-sh)s --prefix=%(prefix)s --openssldir=%(prefix)s/etc/ssl %(autogenargs)s"
              makeinstallargs="install_sw" supports-non-srcdir-builds="no" supports-parallel-builds="no">
-    <branch module="openssl-0.9.8zg.tar.gz" version="0.9.8zg" repo="openssl"/>
+    <branch module="openssl-0.9.8zg.tar.gz" version="0.9.8zg" repo="openssl">
```

Comment:
i decided to use the current versions for now; i'll try newer dependencies some time later.